### ISSUE TITLE
Animations are synchronized across the UI to avoid unbalanced animation frequency.

### ISSUE THREAD

**Issue by senyk** [jryannel/qt5-cadaques#92]
Not sure if this is the right place for that sentence.
Isn't it something for the beginning?
And I would rephrase it:
"All Animations in QtQuick are controlled by the same timer, and are therefor synchronized. This improves the performance and visual quality of animations."

back-link: ch05/index.html#grouped-animations

**Comment by jryannel** [comment]
Nice sentence. Thanks
